.circleci/config.yml

```diff
@@ -184,6 +184,7 @@ jobs:
         auth:
           username: $DOCKERHUB_USERNAME
           password: $DOCKERHUB_PASSWORD
+    parallelism: 4
```

Comment:
🤔

Reply:
yeah 🤔. will this cause issues with integration tests expecting strict database counts/values? or does this mean a whole separate mysql instance is spun up to parallelize the tests? (the [circle docs](https://circleci.com/docs/2.0/parallelism-faster-jobs/) aren't helping me much here 🙃 )

Reply:
Oh, I forgot to amend the commit to mention this. It's from a tip on Circle CI: https://app.circleci.com/pipelines/github/Pocket/curated-corpus-api/2672/workflows/a46f14e1-f21d-430c-8745-79e31174879c/jobs/11765. I thought I'd give it a try and see if anything breaks. It doesn't 🙃. However, it doesn't seem to speed the tests up at all - they still run for ~2 mins, but now in four different containers. I'll merge this one as-is but make a note to take this out in a follow-up PR.

Reply:
Update: nope, I'll do it here since I need to update another part of code anyway.